### PR TITLE
support auth delegating

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: charts/multicluster-controlplane/templates/deployment.yaml [hunk 1]
```diff
@@ -32,13 +32,18 @@ spec:
         {{- if eq .Values.enableSelfManagement true }}
         - "--self-management"
         {{- end }}
+        {{- if eq .Values.enableDelegatingAuthentication true }}
+        - "--delegating-authentication"
```
Comment (clyang82) [inline charts/multicluster-controlplane/templates/deployment.yaml:36]: Can you write a simple document how to use it in OpenShift cluster?

Comment (skeeey) [reply charts/multicluster-controlplane/templates/deployment.yaml:36]: yeah, I will open a new pr to add the document

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**